charts/velero/values.yaml

```diff
@@ -204,6 +204,30 @@ credentials:
   # true and `existingSecret` is empty. This should be the contents
   # of your IAM credentials file.
   secretContents: {}
+  # cloud: |
+  #   [default]
+  #   aws_access_key_id=<REDACTED>
+  #   aws_secret_access_key=<REDACTED>
+  # cloud: |
+  #   AZURE_SUBSCRIPTION_ID=<REDACTED>
+  #   AZURE_TENANT_ID=<REDACTED>
+  #   AZURE_CLIENT_ID=<REDACTED>
+  #   AZURE_CLIENT_SECRET=<REDACTED>
+  #   AZURE_RESOURCE_GROUP=<REDACTED>
+  #   AZURE_CLOUD_NAME=AzurePublicCloud
+  # cloud: |
+  #   {
+  #     "type": "<REDACTED>",
+  #     "project_id": "<REDACTED>",
+  #     "private_key_id": "<REDACTED>",
```

Comment:
Would it be possible to have a comment / note about what is optional vs mandatory? For example are *both* the `private_key_id` and `private_key` needed?

Reply:
NP!

Reply:
For GCP, the _credentials-velero_ is fetched from gcloud CLI, ref to https://github.com/vmware-tanzu/velero-plugin-for-gcp#set-permissions-for-velero. So, I think all are mandatory.

However, I add the available values for `AZURE_CLOUD_NAME`.